docs/userstylesyml.md

```diff
@@ -0,0 +1,154 @@
+<p align="center">
+  <h2 align="center">📖 <code>userstyles.yml</code></h2>
+</p>
+
+<p align="center">
+	A guide to all things <a href="../scripts/userstyles.yml"><code>userstyles.yml</code></a>.
+</p>
+
+&nbsp;
+
+### Table of Contents
+
+<!--toc:start-->
+
+- [Adding a new userstyle](#adding-a-new-userstyle)
+- [Adding yourself as a maintainer](#adding-yourself-as-a-maintainer)
+- [Removing yourself as a maintainer](#removing-yourself-as-a-maintainer)
+
+<!--toc:end-->
+
+### Adding a new userstyle
+**Please note that the `README`s for each port are auto-generated by scripts
+defined in [`scripts/generate/`](../scripts/generate)**, along with many other parts of
+this repository. This file, [`userstyles.yml`](../scripts/userstyles.yml), is responsible for holding the
+necessary data to auto-generate everything.
+
+[`userstyles.schema.json`](../scripts/userstyles.schema.json) is the schema
+specification for this file. The fields in the following instructions are based
+on the schema:
+
+1. Add your name to the list of collaborators.
```

Comment:
Why is there a step here that repeats the same stuff as on line 93? We should link to that section instead I think.

Reply:
Fair argument.